index.html

```diff
@@ -163,6 +213,70 @@ <h2>About</h2>
             </div>
         </div>
     </section>
+
+
+    <section class="form-section">
+      <div class="form-background"></div>
+
+      <div class="get-started"> 
+
+      <h2>Get started</h2>
+  <p>
+    I'm always interested in hearing about <br />
+    new projects, so if you'd like to chat <br />
+    please get in touch.
+  </p>
```

Comment:
- [ ] To improve the alignment of the get started section, it's recommended to adjust the layout of the 'Get Started' section by adding a left margin and ensuring that the font, colors, and spacing match the Figma design. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![yourdesign](https://github.com/Bleu-Yves-Sopoude/mb-portfolio/assets/70483835/e6abdb56-fec3-4308-8cc3-48d2ac4b6ecc) | ![Figma](https://github.com/Bleu-Yves-Sopoude/mb-portfolio/assets/70483835/278f851c-59c4-4a2e-8860-23c1cba8d2fe)